cmd/root.go

```diff
@@ -53,7 +52,15 @@ func RootCmdImp(cmd *cobra.Command, args []string) error {
 // Execute adds all child commands to the root command sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	if err := RootCmd.Execute(); err != nil {
```

Comment:
Adding support for `--param` and `--param-file`, parsing these two CLI options and reading their values into `utils.Flag.params`.